Makefile

```diff
@@ -40,6 +40,12 @@ acceptance-web: build
 test:
 	go test -race -cover ./...
 
+
+.PHONY: test-component
+test-component:
+	go test -race -cover -component | grep -v created_at
```

Comment:
why are you doing this grep?

Reply:
the output from the go test contains a lot of noisy log messages that make it hard to read - the grep gets rid of them

Reply:
We should consider raising a story to modify ONSdigital log to allow functionality similar to log.SetOutput(ioutil.Discard)